### ISSUE TITLE
grad for BN2d

### ISSUE THREAD

**Issue by AlexSunNik** [exiawsh/StreamPETR#89]
Great work! I am wondering whether setting requires_grad = True for BN2d layers of ResNet50 affect the performance? I currently need the gradient on the BN layers for one of our experiments. Thanks!

**Comment by exiawsh** [comment]
It will affect the performance when using small batch size (less than 4 per gpu). I suggest you set it to False
